index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>YOUR NAME HERE</title>
+  <title>Justin Tam</title>
 </head>
 <body>
-
+  <h1>How I feel</h1>
+  <img src="http://i.giphy.com/6XjDsvyFpIhYA.gif">
+  <br><p1>This is what happens when you binge House of Cards
```

Comment:
`br` isn't actually a "self-closing" tag. It's a little weird. You actually have two breaks here, not one!

Rule of thumb is to put `br` tags at the end of the line, or on their own line. Your call.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br